server/Routes/Consumer/Consumer.py

```diff
@@ -0,0 +1,47 @@
+from http import HTTPStatus
+from flask import make_response, request, abort, jsonify
+from flask_restful import Resource, reqparse
+
+from server.Managers.Matches.MatchRequestManager import MatchRequestManager
+from server.config import Configuration
+
+
+class ConsumerRoute(Resource):
+    def check_auth(self):
+        parser = reqparse.RequestParser()
+        parser.add_argument('token', type=str, required=True,
+                            help='Must provide secret token to use this api')
+        data = parser.parse_args()
+        if data['token'] != Configuration.CONSUMER_TOKEN:
+            return abort(HTTPStatus.UNAUTHORIZED)
+
+    def post(self):
```

Comment:
This will happen within the leaderboard, can be disabled and commented out for now.